src/components/Settings/GameBoardSettings.vue

```diff
@@ -27,6 +29,55 @@
       </select>
     </div>
 
+    <div class="setting">
+      <i class="fas fa-bars" />
+      <h5>Header position</h5>
+
+      <select v-model="value[platform.code].position" @change="$emit('save')">
+        <option
+          v-for="{ id, name } in positions"
+          :key="id"
+          :value="id"
+        >
+          {{ name }}
+        </option>
+      </select>
+    </div>
+
+    <div class="setting">
+      <i class="fas fa-shapes" />
+      <h5>Border Radius</h5>
+
+      <toggle-switch
+        id="borderRadius"
+        @change="$emit('save')"
+        v-model="value[platform.code].borderRadius"
+      />
```

Comment:
not a big fan of making the border radius a setting, what do you think about making it a css variable and have each theme determine the radius in pixels? If you go that route, I would make sure it defaults to the current value of `$borderRadius`